### ISSUE TITLE
Design an Info Session

### ISSUE THREAD

**Issue by AdrianCollado** [rcos/intro#32]
We should design an info session for incoming freshmen who might be interested in RCOS.

**Comment by mskmoorthy** [comment]
Good idea (more developers maintainers for yass) -  Do you know how long will it take? Is there any one besides Richie to help out

**Comment by kburk1997** [comment]
See my comment on issue #33.
